src/sentry/static/sentry/app/views/alerts/rules/details/body.tsx

```diff
@@ -80,9 +82,17 @@ export default class DetailsBody extends React.Component<Props> {
 
   getTimePeriod() {
     const {location} = this.props;
+    const now = moment.utc();
 
     const timePeriod = location.query.period ?? TimePeriod.ONE_DAY;
-    return TIME_OPTIONS.find(item => item.value === timePeriod) ?? TIME_OPTIONS[1];
+    const timeOption =
+      TIME_OPTIONS.find(item => item.value === timePeriod) ?? TIME_OPTIONS[1];
+
+    return {
+      ...timeOption,
+      start: getUtcDateString(moment(now.diff(TIME_WINDOWS[timeOption.value]))),
```

Comment:
nice, so the related issues are in the same period as what is shown in the graph?

Reply:
Yes exactly